TerminalDocs/command-line-arguments.md

```diff
@@ -4,33 +4,142 @@ description: Learn how to create command line arguments for Windows Terminal.
+The `wt` command line accepts two types of values: options and commands:
 
-Here are some examples:
+```bash
+wt [options] [command ; ]
+```
 
-`wt -d .`
-Opens the Terminal with the default profile in the current working directory.
+Options are a list of flags and other parameters that can control the behavior of the `wt` command line as a whole. Commands are a semicolon-delimited list of commands and arguments for those commands.
 
-`wt -d . ; new-tab -d C:\ pwsh.exe`
-Opens the Terminal with two tabs. The first is running the default profile starting in the current working directory. The second is using the default profile with pwsh.exe as the "commandline" (instead of the default profile’s "commandline") starting in the C:\ directory.
+If no command is specified, then the command is assumed to be `new-tab` by default. This allows for the following interpretations:
 
-`wt -p "Windows PowerShell" -d . ; split-pane -V wsl.exe`
-Opens the Terminal with two panes, split vertically. The top pane is running the profile with the name “Windows Terminal” and the bottom pane is running the default profile using wsl.exe as the "commandline" (instead of the default profile’s "commandline").
+| Command | Interpretation |
+| ------- | -------------- |
+| `wt cmd.exe` | `wt new-tab cmd.exe` |
+| `wt ; ; ;` | `wt new-tab ; new-tab ; new-tab ; new-tab` |
 
-`wt -d C:\Users\cinnamon\GitHub\WindowsTerminal ; split-pane -p "Command Prompt" ; split-pane -p "Ubuntu" -d \\wsl$\Ubuntu\home\cinnak -H`
+## Command line options
 
-The image below illustrates this command line argument opening the terminal with multiple split panes.
+| Option | Description |
+| ------ | ----------- |
+| `--help`, `-h`, `-?`, `/?` | Displays the help message. |
 
-![Windows Terminal Command Line Argument for Split Panes](./images/terminal-command-args.gif)
\ No newline at end of file
+## Command line commands
+
+### Create a new tab
+
+This opens a new tab with the given customizations. On its first invocation, this also opens a new window. Subsequent `new-tab` commands will all open new tabs in the same window.
+
+```bash
+new-tab [parameters]
+```
+
+#### Parameters
+
+| Parameter | Description |
+| --------- | ----------- |
+[!INCLUDE [terminal-parameters](./terminal-parameters.md)]
+
+<br />
+
+___
+
+### Open a new pane
+
+This opens a new pane with the given customizations in the currently focused tab by splitting the given pane vertically or horizontally.
+
+```bash
+split-pane [-H, --horizontal]|[-V, --vertical] [parameters]
+```
+
+#### Parameters
+
+| Parameter | Description |
+| --------- | ----------- |
+| `-H,--horizontal`, `-V,--vertical` | Used to indicate which direction to split the pane. `-V` is "vertically" (think `[|]`), and `-H` is "horizontally" (think `[-]`). If omitted, defaults to "auto", which splits the current pane in whatever the larger dimension is. If both `-H` and `-V` are provided, defaults to vertical. |
```

Comment:
Just a heads up, the `[|]` and `[-]` in here confuse markdown rendering in github - unsure how it ens up in the final docs site.

Reply:
(I think it's because of the `|` in the table)

Reply:
This one ended up being fine. The `terminal-parameters` one does not though.